utils/AnApiSubmission.ts

```diff
@@ -0,0 +1,60 @@
+const submitANForm = async (
+  email: string,
+  postcode: string,
+  constituency: Constituency,
+  anFormUrl: string,
+  sourceCode: string = "",
+): Promise<Response> => {
+  if (!email || !anFormUrl) {
+    return Response.error();
+  }
+
+  // hack to get around AN constituency name problems
+  const an_name_map: { [index: string]: string } = {
+    "Ynys Môn": "Ynys Mon",
+    "Montgomeryshire and Glyndŵr": "Montgomeryshire and Glyndwr",
+  };
```

Comment:
This is kind of hacky but maybe it's better than having extra fields in the csv etc?

Reply:
Yeah, think for the sake of these 2x constituencies, fine to have a hard-coded mapping... Can always add a TODO to look to improve this, eg. by adding ActionNetwork constituency names to the SQLite, and returning them from the API.